### DISCUSSION TITLE
Chia 2.0.0 with gigahorse

### DISCUSSION THREAD

**Discussion by scoville07** [madMAx43v3r/chia-gigahorse#192] [Q&A]
Hello,

is anyone using chia 2.0.0 already with gigahorse/madmax compressed plots?

If I start chia 2.0.0 it show all my plots invalid. I enabled "compressed plot support", but for example the option "enable gpu harvest" is not even there in the GUI. My farmer has a rtx3060 with 12gb ram and 64GB RAM.

So I still keep using the gigahorse farmer, not a problem. Just out of interest as well, will there be a gigahorse 2.0.0, too?

Thanks
Christian

**Comment by Kruger1981** [comment]
You always need to 1st open gigahorse before opening the UI

Also, some options won't work well with giga14 and GUI 2.0.0

**Comment by NightWolfHome** [comment]
When is the Gigahorse 2.0 release planned?

**Comment by Putas123** [comment]
Hi.
I have gigahorse 200 and GUI 211
I have made compressed plots C8 2000 pieces + C9 300 pieces + C18 350 pieces.
The GUI sees all plots with an access time of 4-5 seconds. Pool only sees 320GB of plots.
4070 Ti graphics card.
What could be the reason?

**Reply by Kruger1981** [reply]
Takes time for the plot count to go up on the pool
